ProjectStructureAndRoles.md

```diff
@@ -75,17 +70,17 @@ A Project participant may resign by notifying the team mailing list and by sendi
 
 **Working Group participants**
 
-Each Project participant can be part of one or more of the Working Groups of the Project by adding his name to the WG_PARTICIPANTS.MD files. By that he gets the role Working Group participant.
+Each Project participant can be part of one or more of the Working Groups of the Project by subscribing to the mailing list(s). By that he gets the role Working Group participant, the WG_PARTICIPANTS.MD file(s) is/are then also adjusted by the administrators of the Project.
 
```

Comment:
By that " he/she" gets the role [...]

Reply:
to be consistent and inclusive I propose "they get"

Reply:
is changed now